### PR TITLE
init new recipe perl-text-csv

### PR DESCRIPTION
**Pull request by fgiacomoni**
* [ X] I have read the guidelines above.
* [X ] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl-text-csv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl-threaded
+
+  skip: True  # [osx]
```
Comment (bgruening) [inline recipes/perl-text-csv/meta.yaml]: If you want to skip osx this needs to be under the general `build` section.

Comment (bgruening) [inline recipes/perl-text-csv/meta.yaml]: But it seems that it works under OSX, so just remove this line :)

Comment (fgiacomoni) [inline recipes/perl-text-csv/meta.yaml]: done in commit 3e07d98 - thanks

File: recipes/perl-text-csv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: perl-text-csv
+  version: "1.33"
+
+source:
+  fn: Text-CSV-1.33.tar.gz
+  url: http://search.cpan.org/CPAN/authors/id/M/MA/MAKAMAKA/Text-CSV-1.33.tar.gz
```
Comment (bgruening) [inline recipes/perl-text-csv/meta.yaml]: Oh sorry and I missed this, can you please also add a checksum?

Comment (fgiacomoni) [inline recipes/perl-text-csv/meta.yaml]: No problem - I'm discovering recipes and missing important points ! ;-)
Done - 701afdf

**Comment by bgruening** [comment]
Awesome!

**Pull request merged**